doc/user/misc.xml

```diff
@@ -177,6 +177,33 @@ SCons 1.0 or greater required, but you have SCons 0.98.5
 
   </section>
 
+  <section>
+  <title>Grabbing the current SCons Version:  the &GetSConsVersion; Function</title>
```

Comment:
To use the static entity like this, it needs to be added to `doc/scons.mod`. However, the *generated* entity `&f-GetSConsVersion;` (f for "function") should work as long as the matching function description is in one of the XML files, which in fact it is - `<scons_function name="GetSConsVersion">` in `SConscript.xml` and you can see it generated in `doc/generated/functions.mod`.  Clear as mud?